<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/images/images/background.png);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn_search{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgb(193, 248, 247), stop:1 rgb(193, 248, 247));
	color:rgba(255, 255, 255, 210);
	border-radius:5px;
}
QPushButton#btn_search:hover{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgb(193, 248, 247), stop:1 rgb(193, 248, 247));
}
QPushButton#btn_search:pressed{	
	padding-left:3px;
	padding-top:3px;
	background-color:rgb(193, 248, 247);
}

QLabel#lbl_location{
	background-color: rgba(0,0,0,0);
}</string>
    </property>
    <widget class="QLineEdit" name="line_city">
     <property name="geometry">
      <rect>
       <x>322</x>
       <y>90</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(105,118,132,255);
color:black;
padding-bottom:7px;</string>
     </property>
     <property name="placeholderText">
      <string>       City</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_search">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>160</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#btn_search{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgb(193, 248, 247), stop:1 rgb(193, 248, 247));
	color:rgba(255, 255, 255, 210);
	border-radius:5px;
}
QPushButton#btn_search:hover{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgb(193, 248, 247), stop:1 rgb(193, 248, 247));
}
QPushButton#btn_search:pressed{	
	padding-left:3px;
	padding-top:3px;
	background-color:rgb(193, 248, 247);
}</string>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_location">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>210</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 255, 255);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_temp">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>310</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_condition">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>410</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_home">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>550</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#btn_home{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(20, 47, 78, 219), stop:1 rgba(85, 98, 112, 226));
	color:rgba(255, 255, 255, 210);
	border-radius:5px;
}
QPushButton#btn_home:hover{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(40, 67, 98, 219), stop:1 rgba(105, 118, 132, 226));
}
QPushButton#btn_home:pressed{	
	padding-left:3px;
	padding-top:3px;
	background-color:rgba(105, 118, 132, 200);
}</string>
     </property>
     <property name="text">
      <string>HOME</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
